.github/workflows/gradle.yml

```diff
@@ -53,6 +54,9 @@ jobs:
       - name: Run Integration Tests
         run: ./gradlew --no-daemon integrationTest --scan
 
+      - name: Run Standard Source Tests
```

Comment:
adding standard source tests to build as these are the only tests that currently run for salesforce and github.